pyiron/atomistics/structure/analyse.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import numpy as np
+from pyiron_base import Settings
+from sklearn.cluster import AgglomerativeClustering
+
+__author__ = "Joerg Neugebauer, Sam Waseda"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Sam Waseda"
+__email__ = "waseda@mpie.de"
+__status__ = "production"
+__date__ = "Sep 1, 2017"
+
+s = Settings()
+
+class Analyse:
```

Comment:
Should inherit from pyiron_base.PyironFactory

Reply:
What does it change?

Reply:
I spoke too soon anyhow. 

At the moment it adds an equivalence between `__getitem__` and `getattr`, which isn't much.

I had originally suggested it because eventually I guess we'll be iterating over factory classes and adding them programmatically depending on the modules which have been imported. But this was stupid of me, because that's just for factories which will belong to the `Creator` class of `Project` -- which this obviously doesn't. So please do not inherit from `PyironFactory` 😛